src/views/accounts/AccountChangeSecret/Executions/AccountChangeSecretExecutionList.vue

```diff
@@ -55,12 +55,12 @@ export default {
             formatterArgs: {
               drawer: true,
               getDrawerTitle({ row }) {
-                return row.snapshot.name
+                return row.automation.name
               },
-              getTitle: ({ row }) => row.snapshot.name,
+              getTitle: ({ row }) => row.automation.name,
               getRoute: ({ row }) => ({
                 name: 'AccountChangeSecretDetail',
-                params: { id: row.automation }
+                params: { id: row.automation.id }
               })
             },
             id: ({ row }) => row.automation
```

Comment:
The code has no obvious inconsistencies. The changes suggested above can be merged seamlessly into the current version of the code without affecting its functionality or readability. There aren't any significant changes needed based on the given details from 2021-09-01 to present day.

In summary:

No major issues identified;
Suggestions not requiring modification at this time; and
Code doesn't need adjustment as it appears consistent with previous versions.